src/app/resources/resources.component.ts

```diff
@@ -1149,7 +1152,54 @@ export class ResourcesComponent implements OnInit, AfterViewInit, OnDestroy {
   scrollToTop() {
     this.contentView.scrollToTop();
   }
-  exploreOtherContents() {
+
+  async requestMoreContent() {
+    const navigationExtras = {
+      state: {
+        subjects: [...this.subjects],
+        categoryGradeLevels: this.categoryGradeLevels,
+        storyAndWorksheets: this.storyAndWorksheets,
+        contentType: ContentType.FOR_LIBRARY_TAB,
+        selectedGrade: this.getGroupByPageReq.grade,
+        selectedMedium: this.getGroupByPageReq.medium
+      }
+    };
+    const req: FormRequest = {
+      type: "dynamicform",
+      subType: "support",
+      action: "get",
+      component: "app"
+    } as any;
+    const formConfig = (await this.formService.getForm(req).toPromise() as any).form.data.fields;
```

Comment:
Remove the Form request from resources component and move it to FormnFrameworkUtilityService